starknet_py/devnet_utils/devnet_client_models.py

```diff
@@ -0,0 +1,94 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from starknet_py.net.client_models import Hash, PriceUnit
+
+
+@dataclass
+class MintResponse:
+    """
+    Represents the result of a mint operation, including the new balance.
+    """
+
+    new_balance: int
+    unit: PriceUnit
+    tx_hash: Hash
```

Comment:
Let's use the specific types we deserialize to, instead of `Hash`. For example here it should be `int`.